caasp-istio-pilot-image/_service

```diff
@@ -1,7 +1,7 @@
 <services>
     <service name="replace_using_package_version" mode="buildtime">
         <param name="file">caasp-istio-pilot-image.kiwi</param>
-        <param name="regex">%%PKG_VERSION%%</param>
+        <param name="regex">1.5.4</param>
```

Comment:
This makes no sense. If for some reason the tag you need does not match the version of any of the installed packages installed within the image, then this service is not needed. If istio is in version `1.5.4` then please use the service and you'll get that automatically aligned on following istio updates without having to submit image source changes just to bump de version.

Reply:
the image will now be reverted to use %%PKG_VERSION%% and the cilium-istio-proxy package is now being modified to build as version 1.5.4. So this hard-coded tagging can be removed and _service file can be retained.